turbolinks/pricing.html

```diff
@@ -0,0 +1,217 @@
+				</div>
+
+				<h2 class="display-6 text-center mb-4">Compare plans</h2>
+
+				<div class="table-responsive">
+					<table class="table text-center">
+						<thead>
+							<tr>
+								<th style="width: 34%;"></th>
+								<th style="width: 22%;">Free</th>
+								<th style="width: 22%;">Pro</th>
+								<th style="width: 22%;">Enterprise</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<th scope="row" class="text-start">Public</th>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+							<tr>
+								<th scope="row" class="text-start">Private</th>
+								<td></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+						</tbody>
+
+						<tbody>
+							<tr>
+								<th scope="row" class="text-start">Permissions</th>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+							<tr>
+								<th scope="row" class="text-start">Sharing</th>
+								<td></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+							<tr>
+								<th scope="row" class="text-start">Unlimited members</th>
+								<td></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+							<tr>
+								<th scope="row" class="text-start">Extra security</th>
+								<td></td>
+								<td></td>
+								<td><svg class="bi" width="24" height="24"><use xlink:href="#check"/></svg></td>
+							</tr>
+						</tbody>
+					</table>
+				</div>
+
+
+			</main>
+		</div>
+	</div>
+
+
+	<script src="./assets/dist/js/bootstrap.bundle.min.js" data-turbolinks-track="reload"></script>
+
+	<script src="https://cdn.jsdelivr.net/npm/feather-icons@4.28.0/dist/feather.min.js" integrity="sha384-uO3SXW5IuS1ZpFPKugNNWqTZRRglnUJK6UAZ/gxOX80nxEkN9NcGZTftn6RzhGWE" crossorigin="anonymous" data-turbolinks-track="reload"></script><script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.4/dist/Chart.min.js" integrity="sha384-zNy6FEbO50N+Cg5wap8IKA4M/ZnLJgzc6w2NqACZaK0u0FXfOWRRJOnQtpZun8ha" crossorigin="anonymous" data-turbolinks-track="reload"></script><script src="dashboard.js"></script>
+
+	<script src="https://cdn.jsdelivr.net/npm/turbolinks@5.2.0/dist/turbolinks.min.js"></script>
```

Comment:
Whenever we reference a script on a CDN we should include:
* an integrity tag. The CDN should have an option with a link to get the integrity tag
* crossorigin="anonymous"